### PR TITLE
Added monkey status codes

### PR DESCRIPTION
**Pull request by Miodec**

### CODE REVIEW

File: backend/api/controllers/result.js [hunk 1]
```diff
@@ -60,6 +61,10 @@ class ResultController {
     const { uid } = req.ctx.decodedToken;
     const { result } = req.body;
     result.uid = uid;
+    if (result.acc !== 100) {
```
Comment (Bruception) [inline backend/api/controllers/result.js:64]: Results without 100% accuracy will fail?

File: backend/constants/monkey-status-codes.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+const statuses: Statuses = {
+  RESULT_TOO_SHORT: {
+    code: 461,
+    message: "Test too short",
+  },
+  GIT_GUD: {
+    code: 469,
+    message: "Git gud scrub",
```
Comment (Bruception) [inline backend/constants/monkey-status-codes.ts:22]: Lol

**Review by Bruception — DISMISSED** [review]
LGTM IIW, just a question

File: backend/api/controllers/result.js (hunk 1 above)
Comment (Miodec) [reply backend/api/controllers/result.js:64]: Just for testing

File: backend/constants/monkey-status-codes.ts (hunk 2 above)
Comment (Miodec) [reply backend/constants/monkey-status-codes.ts:22]: :)

**Review by Bruception — APPROVED** [review]
RE-LGTM IIW

**Review by Ferotiq — APPROVED** [review]

**Comment by Ferotiq** [comment]
late to the party

**Comment by Miodec** [comment]
image

**Pull request merged**